contracts/src/test/7683/T1BasicSwapE2E.t.sol

```diff
@@ -697,26 +774,38 @@ contract T1BasicSwapE2E is BaseTest {
         assertEq(balancesAfterRefund[balanceId[kakaroto]], balancesBeforeRefund[balanceId[kakaroto]] + amount);
     }
 
-    function _handleRelayMessage(bytes32[] memory orderIds, bytes[] memory ordersFillerData, bool isSettle) internal {
+    function _handleRelayMessage(
+        bytes32[] memory orderIds,
+        bytes[] memory ordersFillerData,
+        bool isSettle,
+        uint32 _destination,
+        uint256 batchIndex
+    )
+        internal
+    {
         rollup.addProver(address(0));
         bytes memory batchHeader1 = BatchHeaders.generateBatchHeader(rollup);
         assertEq(rollup.isBatchFinalized(1), false);
 
         bytes memory innerMessage = abi.encode(isSettle, orderIds, ordersFillerData);
 
-        bytes memory outerMessage =
-            abi.encodeWithSelector(T1ERC7683.handle.selector, destination, destinationRouterB32, innerMessage);
-
         // hash 0xcca132db240c06c148d210ceda18701a38e863e5ab2ed4638b15b6c7b30a08ae
         uint256 nonce = 0;
         uint256 msgValue = 0;
         address from = address(destinationRouter);
         address to = address(originRouter);
+        bytes memory proof = hex"";
```

Comment:
could we do tests a couple of proofs at different heights? perhaps up to height 2